src/app/Environment.java

```diff
@@ -0,0 +1,28 @@
+package app;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ *
+ * @author matt cano
+ */
+public class Environment {
+
+    private String env = null;
+
+    public Environment() {
+        try (InputStream in = new FileInputStream("foodmood.properties")) {
```

Comment:
An exception is being thrown here. I'm not sure why. It causes getEnvironment to return null, which then causes MoodDao to cause and exception. If it's weren't for the MoodDao involvement this wouldn't be blocking but this issue does actually break the app.

There are two changes I suggest. Either would fix the issue but both are quick and provide more protection.
1. Don't set env to null. Use "" instead.
2. In MoodDao, test for null along with "development".